lib/openstack_api/openstack_server.py

```diff
@@ -16,8 +17,15 @@ def snapshot_and_migrate_server(
     :param conn: Openstack Connection
     :param server_id: Server ID to migrate
     :param server_status: Status of machine to migrate - must be ACTIVE or SHUTOFF
+    :param flavor_name: Server flavor name
     :param dest_host: Optional host to migrate to, otherwise chosen by scheduler
     """
+
+    if flavor_id.startswith("g-"):
+        raise ValueError(
+            f"Attempted to move GPU flavor, {flavor_id}, which is not allowed!"
+        )
```

Comment:
We should use an `if any(flavour_id.startswith(i) for i in BLOCKED_PREFIXES)" then we can easily add values since I think some other flavours (`c-`) need to be added here too